python/llm/example/CPU/HF-Transformers-AutoModels/Model/qwen-vl/chat.py

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from bigdl.llm.transformers import AutoModel, AutoModelForCausalLM
+from transformers import AutoTokenizer, LlamaTokenizer
+from transformers.generation import GenerationConfig
+import torch
+import time
+import os
+import argparse
+from bigdl.llm import optimize_model
+torch.manual_seed(1234)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Predict Tokens using `chat()` API for Qw-VL-Chat model')
+    parser.add_argument('--repo-id-or-model-path', type=str, default="Qwen/Qwen_VL_Chat",
+                        help='The huggingface repo id for the Qwen_VL_Chat model to be downloaded'
+                             ', or the path to the huggingface checkpoint folder')
+    parser.add_argument('--n-predict', type=int, default=32, help='Max tokens to predict')
+    
+    current_path = os.path.dirname(os.path.abspath(__file__))
+    args = parser.parse_args()
+    model_path = args.repo_id_or_model_path  
+        
+    # Load model
+    # For successful BigDL-LLM optimization on Qwen-VL-Chat, skip the 'c_fc' and 'out_proj' modules during optimization
+    model = AutoModelForCausalLM.from_pretrained(model_path, load_in_4bit=True, device_map="cpu", trust_remote_code=True, optimize_model=True,modules_to_not_convert=['c_fc', 'out_proj'] )
```

Comment:
Maybe also change the alignment here:

```python
model = AutoModelForCausalLM.from_pretrained(model_path, 
                                             load_in_4bit=True, 
                                             device_map="cpu",
                                             trust_remote_code=True, 
                                             modules_to_not_convert=['c_fc', 'out_proj'] )
``` 

Also `optimize_model` arg can be removed here as it is True by default.